docs/develop/one_dragon/modules/application_plugin_system.md

```
@@ -0,0 +1,290 @@
+
+**文件位置**: `src/one_dragon/base/operation/application/application_factory.py`
+
+**构造参数**:
+- `app_id`: 应用唯一标识符
+- `app_name`: 显示名称
+- `default_group`: 是否属于默认应用组（一条龙运行列表），默认为 `True`
+- `need_notify`: 是否需要通知，默认为 `False`
+
+## 目录结构
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)